Sources/Danger/Danger.swift

```diff
@@ -14,7 +14,11 @@ final class DangerRunner {
 
     let logger: Logger
     let dsl: DangerDSL
-    var results = DangerResults()
+    var results = DangerResults() {
+        didSet {
+            dump()
+        }
+    }
 
     private init() {
         let isVerbose = CommandLine.arguments.contains("--verbose") || (ProcessInfo.processInfo.environment["DEBUG"] != nil)
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 12...
  
  
  DangerID: danger-id-default;
  File: Sources/Danger/Danger.swift;
  Line: 24;
-->

- :warning: Line should be 120 characters or less: currently 124 characters (`line_length`)